app/views/pages/faq.html.erb

```diff
@@ -3,5 +3,140 @@
 <% cache do %>
 <div class="container">
   <h1 class=about_h1>FAQ</h1>
+  <p class="about_p">Have a questions about the Archives Unleashed Cloud? Check out some of our most frequently asked questions.</p>
+
+    <div id="auk_faq_accordionContainer accordion">
+      <div class="auk_deriv_accord card">
+        <div class="auk_deriv_accord-header card-header" id="headingOne">
+          <h5 class="mb-0">
+            <button class="btn btn-link" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+              AUK Account and Dashboard
+            </button>
+          </h5>
+        </div>
+        <div id="collapseOne" class="collapse show" aria-labelledby="headingOne" data-parent="#accordion">
+          <div class="auk_faq_accord_body card-body">
+            <p class="about_p"><strong>How do I use the Archives Unleashed Cloud?</strong></p>
+            <p class="about_p">If you're new to the Cloud, our <%=link_to('AUK documentation','/documentation') %> page gives a full walkthrough on how to set up and use your account and dashboard.</p>
+            <p class="about_p"><strong>I don't have an Archive-It account! Can I still use the Archives Unleashed Cloud?</strong></p>
+            <p class="about_p">We currently only support <%= link_to('Archive-It', 'https://archive-it.org/', target: '_blank') %> accounts, as our system is based on the Web Archiving Systems API, or <%= link_to('WASAPI', 'https://github.com/WASAPI-Community/data-transfer-apis', target: '_blank') %>. In the future, we are hoping to expand the services that we support. If you have a web archiving service and would like to integrate with the Cloud, <%= link_to('drop us a line', 'mailto:sam.fritz@archivesunleashed.org', target: '_blank') %>.</p>
+            <p class="about_p"><strong>Can I can sync my Archive-It account with the Archives Unleashed Cloud without using Twitter or Github?</strong></p>
+            <p class="about_p">Right now authentication is only offered via <%= link_to('Twitter', 'https://twitter.com/', target: '_blank') %> and <%= link_to('GitHub', 'https://github.com/', target: '_blank') %> - we hope to expand this in the future. We've had some of our colleagues come to the Cloud without an account for either Twitter or GitHub, and if you're in a similar situation we have recommended creating  a research based Twitter/Github account in order to use the Cloud. Don't worry – you don't need to code or tweet!</p>
```

Comment:
"we have recommended creating a research based Twitter/Github account in order to use the Cloud."
->
"we recommend creating a research-based account in order to use the Cloud."